tests/publicLinkTest.js

```diff
@@ -49,11 +52,14 @@ describe('oc.publicFiles', function () {
 
   const folderInPublicShareInteraction = (provider, description, method, statusCode) => {
     return provider
+      .given('the user is recreated', { username: testUser, password: testUserPassword })
+      .given('folder exists', { username: testUser, password: testUserPassword, folderName: testFolder })
+      .given('resource is shared', { username: testUser, password: testUserPassword, resource: testFolder, shareType: 3, permissions: 15 })
       .uponReceiving(description)
       .withRequest({
         method: method,
-        path: MatchersV3.regex(
-          `.*\\/remote\\.php\\/dav\\/public-files\\/${config.shareTokenOfPublicLinkFolder}\\/foo`,
+        path: MatchersV3.fromProviderState(
+          '/remote.php/dav/public-files/${token}/foo', /* eslint-disable-line no-template-curly-in-string */
```

Comment:
does the var `token` get replaced this way? why not using ``backtick(`)``?

Reply:
Yeah, that's how the `fromProviderState` works.